cosmoz-moment-timeago.html

```diff
@@ -5,48 +5,62 @@
 </dom-module>
 
 <script type="text/javascript">
-	(function () {
+	/*global Cosmoz*/
+	const locale = 'en';
```

Comment:
Maybe name it in all caps for a global constant? LOCALE

Reply:
Pushed a change.